src/Database/PostgreSQL/LibPQ.hs

```diff
@@ -1630,6 +1631,24 @@ setSingleRowMode :: Connection
 setSingleRowMode connection = enumFromConn connection c_PQsetSingleRowMode
 
 
+-- | Select chunked mode for the currently-executing query.
+--
+-- This function is similar to 'PQsetSingleRowMode', except that it specifies
+-- retrieval of up to @chunkSize@ rows per 'Result', not necessarily just one
+-- row. This function can only be called immediately after 'sendQuery' or one
+-- of its sibling functions, before any other operation on the connection such
+-- as 'consumeInput' or 'getResult'. If called at the correct time, the
+-- function activates chunked mode for the current query and returns 1.
```

Comment:
> returns 1

But is `IO Bool`?

same for `setSingleRowMode`.

Reply:
I was looking at the same but setSingleRowMode said 0 and 1 too, so I assumed it's OK.

I'll add a commit to change both.